src/IceFloeTracker.jl

```diff
@@ -71,7 +71,7 @@ include("branch.jl")
 include("special_strels.jl")
 include("tilingutils.jl")
 include("histogram_equalization.jl")
-include("imadjust.jl")
```

Comment:
I think the CI fails because of this deletion. Maybe keep the `include("imadjust.jl")` and add include("brighten.jl")? It looks like there is still an `imadjust.jl` script, suggesting that maybe we need both?

Reply:
Must have done that trying to resolve merge conflicts. I think it's fixed now. Thanks.